cmd/eelbot/.gitignore

```diff
@@ -0,0 +1 @@
+eelbot
```

Comment:
is this intentional?

Reply:
Yeah. When I build the eelbot exe in this folder, it gets tracked by git. On windows, it would have the .exe extension so it would get caught by the ignore rule defined in the main .gitignore. However, on linux, exes have no extension, so this seems to be the easiest way to ignore it.